showcase/src/app/demo-ui/pages/dropdown/demo-dropdown-page.component.html

```diff
@@ -126,19 +142,20 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		filesPath="dropdown/white"
 		[exampleTitle]="'SHOWCASE.DEMO.DROPDOWN.WHITE' | translate"
 	>
-		<stark-dropdown
-			class="stark-full-width"
-			color="white"
-			dropdownId="serviceWhiteDropdown"
-			[options]="serviceDropdownOptions"
-			[value]="selectedServiceWhiteDropdown"
-			placeholder="SHOWCASE.DEMO.DROPDOWN.SELECT_SERVICE"
-			optionIdProperty="id"
-			optionLabelProperty="value"
-			(selectionChanged)="whiteDropdownOnChange($event)"
-			required
-		>
-		</stark-dropdown>
+		<mat-form-field class="stark-full-width">
+			<stark-dropdown
+				color="white"
+				dropdownId="serviceWhiteDropdown"
+				[options]="serviceDropdownOptions"
+				[value]="selectedServiceWhiteDropdown"
+				[placeholder]="'SHOWCASE.DEMO.DROPDOWN.SELECT_SERVICE' | translate"
+				optionIdProperty="id"
+				optionLabelProperty="value"
+				(selectionChanged)="whiteDropdownOnChange($event)"
+				required
```

Comment:
Same issue here: it is not working as expected... although it is required, the dropdown is not marked in red when there is no value selected....